packages/rocketchat-livechat/imports/server/rest/visitors.js

```diff
@@ -30,3 +30,35 @@ RocketChat.API.v1.addRoute('livechat/visitor/:visitorToken/room', { authRequired
 		return RocketChat.API.v1.success({ rooms });
 	}
 });
+
+RocketChat.API.v1.addRoute('livechat/visitor/:visitorToken/room/close', { authRequired: true }, {
+	post() {
+		if (!RocketChat.authz.hasPermission(this.userId, 'view-livechat-manager')) {
+			return RocketChat.API.v1.unauthorized();
+		}
+
+		const visitor = LivechatVisitors.getVisitorByToken(this.urlParams.visitorToken);
+		if (!visitor) {
+			return RocketChat.API.v1.failure('Visitor not found');
+		}
+
+		const rooms = RocketChat.models.Rooms.findOpenByVisitorToken(this.urlParams.visitorToken).fetch();
```

Comment:
To close a Livechat Room, the `roomId` needs to be known.
I suggest passing the "roomId" by parameter.